lib/optimizely.rb

```diff
@@ -84,20 +84,20 @@ def initialize(datafile, event_dispatcher = nil, logger = nil, error_handler = n
       @notification_center = NotificationCenter.new(@logger, @error_handler)
     end
 
-    # Activates an A/B test for a user, deciding whether they qualify for the experiment, bucketing them
-    # into a variation if they do, and sending an impression event to Optimizely.
+    # Activates an A/B test for a user, determines whether they qualify for the experiment, buckets a qualified
+    # user into a variation, and sends an impression event to Optimizely.
     #
     # This method takes into account the user `attributes` passed in, to determine if the user
     # is part of the audience that qualifies for the experiment.
     #
     # For more information, see https://github.com/optimizely/ruby-sdk/blob/master/lib/optimizely.rb.
     #
-    # @param experiment_key [string]  The key of the experiment for which to activate the variation.
-    # @param user_id        [string]  The ID of the user for whom to activate the variation.
+    # @param experiment_key [string]  The key of the variation's experiment to activate.
```

Comment:
This shoud just hbe "The experiment to activate". It doesn't make sense to refer to the "variation's experiment".